src/index.html

```diff
@@ -1,14 +1,36 @@
 <!doctype html>
 <html lang="en">
+
   <head>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Moyo header</title>
+    <title> </title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <div class="container">
+      <div class="logo"><img src="images/logo.png" alt="logo"></div>
+      <ul class="navbar_menu">
+        <li class="for_line">
+          <a href="#" class="menu_1">Apple</a>
```

Comment:
Need to fix classes everywhere, also add the nav tag here
```suggestion
    <nav class="nav">
      <ul class="nav__list">
        <li class="nav__item">
          <a href="#" class="nav__link">Apple</a>
```